### ISSUE TITLE
Inconsistent model output across training datasets

### ISSUE THREAD

**Issue by Asad-Shahid** [google-deepmind/open_x_embodiment#7]
For the bridge dataset, the model output is explicitly de-normalized before being compared with the ground truth in the inference notebook. For other datasets, e.g. Viola, and Berkley cable routing, the raw model predictions match quite well with the ground truth.  Does this mean the ground truth actions in these datasets are normalized?

**Comment by kpertsch** [comment]
@quanvuong can you please provide details on the normalization used during training of RT-X?
